releasetasks/templates/funsize.yml.tmpl

```diff
@@ -0,0 +1,175 @@
+{% for partial_version in partial_updates.keys() %}
```

Comment:
Should this loop be after the first task ? That looks like the 'en-US build' equivalent.